app/scripts/app.js

```diff
@@ -481,6 +481,68 @@ angular
       LabelFilter.readPersistedState();
     });
   })
+  .run(function($location, $rootScope, $uibModal) {
+    if ($location.path() === '/logout' || $location.path() === '/error') {
+      return false;
+    }
+    var isLocalLogoutModalShown = false;
+    var updateLastInteraction = function() {
+      localStorage.setItem('lastInteraction', new Date());
+    };
+    var updateLogOut = function(bool) {
+      localStorage.setItem('logOut', bool);
```

Comment:
logOut and closeAllModals aren't specific enough keys, even if you append origin-web-console- to them, it needs to be something like origin-web-console-inactivity-logout or something

that said, im not sure i want to track either of these, see other comments below